client/src/react/components/SetScore.js

```diff
@@ -5,14 +5,16 @@ function Score({ score }) {
   return <div className="set-score__score">{score ? score : 0}</div>;
 }
 
-function SetScore({score}) {
+function SetScore({ index, set }) {
+  const { p1, p2 } = set;
```

Comment:
also `player1` once the match data is updated :)

Reply:
right :D